tests/nested/manual/core20-early-config/install

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# rsyslog.service is expected to be a symlink at this point, and apparmor
+# doesn't control readlink, so this should work even without devmode or
+# system-files plug.
+readlink /etc/systemd/system/rsyslog.service > "$SNAP_COMMON"/debug.txt 2>&1 || true
```

Comment:
very tricky :-)

Reply:
Yes, I was scratching my head over this when working on this test, because I totally expected devmode to be required, yet it worked without it.

Reply:
Actually this is because the readlink syscall is in the default seccomp profile applied to snaps, see https://github.com/snapcore/snapd/blob/master/interfaces/seccomp/template.go#L310

Reply:
(and we don't filter seccomp args for readlink, so like stat, this just works everywhere because running readlink everywhere isn't considered a security vulnerability iiuc)